configs/train_config.yaml

```diff
@@ -8,36 +8,40 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
+## Input files
+noisybw: "None"
```

Comment:
since we're now converging on the `snake_case` pattern for arguments, what do you think of updating `noisybw` to `noisy_bw` and similar for related ones?

Reply:
noisybw seems small enough to be without need of underscores ?

Reply:
that way it matches the naming convention followed by others. right now we have some options with `_` and some without it

Reply:
agree. I'll pick it up as a separate PR.

Reply:
resolving this